dask_planner/src/parser.rs

```diff
@@ -333,6 +400,21 @@ impl<'a> DaskParser<'a> {
                         // use custom parsing
                         self.parse_drop_model()
                     }
+                    "schema" => {
+                        // move one token forward
+                        self.parser.next_token();
+                        // use custom parsing
+
+                        let if_exists = self.parser.parse_keywords(&[Keyword::IF, Keyword::EXISTS]);
+
+                        let schema_name = self.parser.parse_identifier();
+
+                        let drop_schema = DropSchema {
+                            schema_name: schema_name?.value,
```

Comment:
minor nit but I think it is better to have the `?` after the function call

```suggestion
                        let schema_name = self.parser.parse_identifier()?;

                        let drop_schema = DropSchema {
                            schema_name: schema_name.value,
```